shell/platform/embedder/tests/embedder_config_builder.cc

```diff
@@ -110,6 +115,21 @@ void EmbedderConfigBuilder::SetSoftwareRendererConfig(SkISize surface_size) {
   context_.SetupOpenGLSurface(surface_size);
 }
 
+void EmbedderConfigBuilder::SetOpenGLFBOCallBack() {
+  // SetOpenGLRendererConfig must be called before this.
+  FML_CHECK(renderer_config_.type == FlutterRendererType::kOpenGL);
+  renderer_config_.open_gl.fbo_callback = [](void* context) -> uint32_t {
+    FlutterFrameInfo tmp;
```

Comment:
Zero initialize structs for good measure. That makes it resilient to updates to the struct. So, `FlutterFrameInfo frame_info = {};` I'd also avoid vars with terse names like `tmp`.

Reply:
Ditto about the zero initializing structs and ivar names elsewhere.

Reply:
Renamed to `frame_info`, and zero initialized it.